divisible.rb

```diff
@@ -0,0 +1,14 @@
+#using each method
+(1..100).each do 
+    #insert block with argrument to see if number is divisible by 2,3, or 5
+      |number|
+      if number % 2==0
+       puts"#{number} is divisible by 2"
+      
+
+    elsif number %3 ==0 
+        puts"#{number} is divisible by 3"
+            else number % 5==0
+            puts"#{number} is divisible by 5"
+        end
+        end
```

Comment:
The inconsistent spacing makes it harder to read the code. Paying attention to the spacing isn't critical for the assignment, but it helps a lot.

```suggestion
    elsif number % 3 == 0 
        puts"#{number} is divisible by 3"
    else number % 5 == 0
        puts"#{number} is divisible by 5"
    end
end
```

There are Ruby tools that automate this, and I can share them with you if you'd like.